mako/util.py

```diff
@@ -257,9 +257,12 @@ def parse_encoding(fp):
         m = _PYTHON_MAGIC_COMMENT_re.match(line1.decode("ascii", "ignore"))
         if not m:
             try:
-                import parser
```

Comment:
AFAICT "ast.parse" is available in Python 2.7 and all Python 3 versions:

https://docs.python.org/3.8/library/parser.html#parser.suite

"From Python 2.5 onward, it’s much more convenient to cut in at the Abstract Syntax Tree (AST) generation and compilation stage, using the ast module."

then in the 3.9 version the parser suite is deprecated entirely